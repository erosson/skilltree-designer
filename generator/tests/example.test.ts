import { describe, expect, test } from '@jest/globals'
import ex from '../src/example'

test('example output', () => {
    expect(ex).toMatchSnapshot()
})