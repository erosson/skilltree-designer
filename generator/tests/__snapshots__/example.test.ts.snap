// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example output 1`] = `"<svg xmlns="http://www.w3.org/2000/svg" viewBox="-31 -5 62 29"><circle cx="-17" cy="10" r="3"></circle><circle cx="-17" cy="0" r="3"></circle><circle cx="-26" cy="5" r="3"></circle><circle cx="-22" cy="19" r="3"></circle><circle cx="-8" cy="5" r="3"></circle><circle cx="17" cy="10" r="3"></circle><circle cx="17" cy="0" r="3"></circle><circle cx="8" cy="5" r="3"></circle><circle cx="12" cy="19" r="3"></circle><circle cx="26" cy="5" r="3"></circle><line x1="-17" y1="0" x2="-26" y2="5" stroke="black"></line><line x1="17" y1="0" x2="8" y2="5" stroke="black"></line><line x1="-17" y1="0" x2="17" y2="0" stroke="black"></line><line x1="-26" y1="5" x2="8" y2="5" stroke="black"></line></svg>"`;
